tools/pipelines/templates/include-test-stability.yml

```
@@ -136,36 +139,27 @@ jobs:
                 workingDir: ${{ parameters.buildDirectory }}
                 customCommand: 'run ${{ taskTestStep }}'
               condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
-
-      # Test - Upload coverage results
-      # Some webpacked file using externals introduce file name with quotes in them
-      # and Istanbul's cobertura reporter doesn't escape them causing parse error when we publish
-      # A quick fix to patch the file with sed. (See https://github.com/bcoe/c8/issues/302)
-      - ${{ if eq(variables['testCoverage'], true) }}:
+        # Rename the failed tinylicious log
```

```suggestion
        # Rename the tinylicious log file, only if tests failed
```
I think this makes it a bit clearer that the existence of the tinylicious log does not necessarily mean there were failures.